src/dibbs_text_to_code/configs.py

```diff
@@ -21,3 +21,6 @@
     "lab_value": {},
     "lab_interp": {},
 }
+
+MODEL_NAME = "Qwen/Qwen3-Embedding-8B"
```

Comment:
Is this the model we landed on, or was it one of the 'snowflake' models?

Reply:
That's one of the models we're looking at, definitely fine as a placeholder until we figure out the final model